_tools/makefiles/base-image-Makefile

```diff
@@ -1,24 +1,27 @@
-# base-image-Makefile
+#-e base-image-Makefile
 
 IMAGE_BASE_NAME = $(shell basename $(shell pwd))
 IMAGE_TYPE = $(shell basename $(shell dirname $(shell pwd)))
 IMAGE_REGISTRY = $(shell ../_tools/makefiles/get_default_registry.sh)
 IMAGE_FULL_NAME = $(IMAGE_REGISTRY)$(IMAGE_TYPE)/$(IMAGE_BASE_NAME)
 CHILDREN = $(shell ../_tools/relational_sorting.php)
 
+ARM64_CONTEXT = $(shell [ "`uname -m`" = "x86_64" ] && echo "arm" || echo "default" )
```

Comment:
user@nfq002128:~/.tmp
$ [ "`uname -m`" = "x86_64" ] && echo "arm" || echo "default"
arm

is it how it should be?